cluster/store/db.go

```diff
@@ -145,10 +173,14 @@ func (db *localDB) UpdateShardStatus(cmd *command.ApplyRequest, schemaOnly bool)
 	}
 
 	return db.apply(
-		cmd.GetType().String(),
-		func() error { return nil },
-		func() error { return db.store.UpdateShardStatus(&req) },
-		schemaOnly)
+		applyOp{
+			op:                 cmd.GetType().String(),
+			updateSchema:       func() error { return nil },
+			updateStore:        func() error { return db.store.UpdateShardStatus(&req) },
+			schemaOnly:         schemaOnly,
+			applyDbUpdateFirst: true,
```

Comment:
**question (non-blocking):** double checking, is meant to be true ?

Reply:
Yup